pytest_reportportal/service.py

```diff
@@ -231,6 +266,79 @@ def _stop_if_necessary(self):
         except queue.Empty:
             pass
 
+
+    @staticmethod
+    def _add_item_hier_parts_dirs(item, hier_flag, dirs_level, report_parts, dirs_parts, rp_name=""):
+
+        parts_dirs = PyTestServiceClass._get_item_dirs(item)
+        dir_path = item.fspath.new(dirname="", basename="", drive="")
+        for dir_name in parts_dirs[dirs_level:]:
```

Comment:
do we need to do all this action below inside this loop if hier_flag == False?

Reply:
Yes, we use it to accumulate `rp_name`